toggles/webapp/toggles.ts

```diff
@@ -121,11 +118,14 @@ const toggles = {
     {
       id: 'abTestTestTest',
       title: 'Testing the A/B test toggler',
+      type: 'test',
       range: [0, 100],
-      when: request => {
-        return !!request.uri.match(/\/works\/.*$/);
-      },
+    },
+    {
+      id: 'newTags',
+      title: 'A/B test for new tags',
       type: 'test',
+      range: [0, 100],
     },
```

Comment:
is `when` not required anymore? (asking as it's still in the README)

Reply:
Good point – it's required for the cache, but not for the toggles webapp. I'll update the README